owncloud/phoenix
# Added error handling

## Description

First part: Add error handling for shares
Second part: Filter sharee and display the owner correclty

## Related Issue




- Fixes #1252 

## How Has This Been Tested?



- Share a file/folder with User A
- Log in as user A
- Share the same file/folder back to user B
- check if an error message got rendered

## Types of changes


## Checklist:



## Open tasks:

- [ ] detect share owner correctly
- [ ] acceptance tests
